### ISSUE TITLE
If UAC is set to 34%, NVDA can't interract with the UAC prompt dialog

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#7825]
A friend of mine has reported something strange - that her installed copy of NVDA 2017.3 does not read UAC prompts to her, even though as I said it is an installed copy and it is set to run on secure screens and after user sign in. And besides that, NVDA should read UAC prompts even if it is not set to run on secure screens and after user sign in - I can confirm that it does so on my mother's laptop. I used Team Viewer to check what's going on and was surprised to see that the slider for UAC in Control Panel is set not to its default value of 67%, but a level lower - 34%. After setting it to 67%, NVDA started again to read the UAC prompts. Now I've checked this on my machine with the latest "next" snapshot of NVDA as well and can confirm the same behavior - if UAC slider is set to 34% instead of the default 67% one, NVDA stays silent in the UAC prompts, even though I can confirm that the system focus moves in that dialog with TAB and the arrow keys. When UAC is set to 100%, NVDA also does read its prompts. When the UAC slider is set to 34%, the dialog appears on the screen, but the other part of the screen does not get obscured like it happens if the slider is set to 67% or 100%, which is some sort of difference that may be one of the causes for the issue at hand.

### Steps to reproduce:
1. Open Control Panel.
2. Find and open "User Accounts".
3. Navigate to the link "Change UAC settings" and activate it with ENTER.
4. Navigate to the UAC slider and set it to 34%, if it is not set to that value already.
5. Activate the "Continue" button and confirm with "Yes" the change in the UAC prompt.
6. Force the UAC prompt to appear (e.g. by launching a program of your choice with administrative privileges from its context menu).
7. When the UAC prompt appears, listen what NVDA will read and then navigate in the dialog with TAB and/or the arrow keys.

### Expected behavior:
NVDA should read the UAC prompt when it appears and its controls when we navigate between them with the TAB and arrow keys.

### Actual behavior:
NVDA only says "User Account Control" when the UAC prompt appears and stays silent when we navigate between the controls in the dialog.

### System configuration:
- Operating system: Windows 10 Pro version 1709 (build 16299.98), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: next-14685,bc898bdc, installed.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: Palit GeForce GT 630, 1024MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

**Comment by fernando-jose-silva** [comment]
I also have this problem.

**Comment by k-kolev1985** [comment]
I've done some more tests however and seams that other screen readers like Narrator and JAWS also exhibit this behavior. So, it probably is a bug in Windows itself and must be reported to Microsoft.

**Comment by ehollig** [comment]
Duplicate of #6236
